Development.md

```diff
@@ -0,0 +1,28 @@
+# Development Environment
+
+## Prerequisite
+Node is required for local development.
+We recommend using a Node version manager ([nvm](https://github.com/nvm-sh/nvm) for example) to ensure a stable development environment is used. 
+Below are the required node / npm versions:
+```sh
+    "node": "18.13.0",
+    "npm": "8.19.3"
+````
+
+## Installation
+In your working directory, open terminal and paste the following commands:
+```sh
+    git clone https://github.com/WebKit/Speedometer.git
+    cd Speedometer
+    npm install
+````
+
+## Run Development Server
+1. In your terminal run:
+    ```sh
+        npm run dev
+    ````
+2. Open your browser of choice and navigate to [http://127.0.0.1:7000](http://127.0.0.1:7000)
```

Comment:
Locally `npm run dev` already opens the browser for me. Is that an option on the http-server CLI? I'd prob prefer to match what's described here and not auto open, but if we want to keep the current behavior then let's remove this text

Reply:
Great callout - I'll remove the flag in the script